packages/protocol/contracts/stability/MedianDeltaBreaker.sol

```diff
@@ -22,13 +22,14 @@ contract MedianDeltaBreaker is IBreaker, UsingRegistry {
   /* ==================== State Variables ==================== */
 
   // The amount of time that must pass before the breaker can be reset for an exchange.
+  // Should be set to 0 to force a manual reset.
```

Comment:
Could potentially want a constant (like `REQUIRE_MANUAL_RESET = 0`) to avoid magic numbers.

Reply:
The cooldown time isn't read in this contract, I'm not sure how introducing this constant would help.